src/components/Facility/PatientNoteCard.tsx

```diff
@@ -75,14 +76,16 @@ const PatientNoteCard = ({
     });
     if (res?.status === 200) {
       Success({ msg: "Note updated successfully" });
+      queryClient.invalidateQueries({
+        queryKey: ["notes", patientId],
+        exact: false,
```

Comment:
exact is false by default
```suggestion
        queryKey: ["notes", patientId],
```